tests/jballerina-debugger-integration-test/src/test/java/org/ballerinalang/debugger/test/adapter/variables/VariableVisibilityTest.java

```diff
@@ -283,6 +282,9 @@ public void localVariableVisibilityTest() {
         debugTestRunner.assertVariable(localVariables, " /:@[`{~π_var", "IL with special characters in var", "string");
         debugTestRunner.assertVariable(localVariables, "üňĩćőđę_var", "IL with unicode characters in var", "string");
         debugTestRunner.assertVariable(localVariables, "ĠĿŐΒȂɭ_ /:@[`{~π_json", "map<json> (size = 0)", "json");
+
+        // service variable visibility test
+        debugTestRunner.assertVariable(localVariables, "serviceVar", "service", "object");
```

Comment:
The visible type name for the service variable should be `service`

Reply:
Fixed with https://github.com/ballerina-platform/ballerina-lang/pull/31139/commits/07acddea6a25135d31ed62ee839b34143d9c0f36